demo/Tables/obs4MIPs_CV.json

```diff
@@ -1,231 +0,0 @@
-{
-    "CV":{
```

Comment:
From the looks of all the red in this PR, it would seem that you have killed most of the contents of the demo directory

Reply:
Yep, hunch confirmed my the PR email alert, "D" means deleted, "M" means modified
![deaddemodir](https://user-images.githubusercontent.com/3229632/30940144-e3ede1bc-a394-11e7-901f-dd7602a73add.png)